ledger/src/blockstore_meta.rs

```diff
@@ -344,7 +344,7 @@ impl ErasureMeta {
 
     // Returns true if the erasure fields on the shred
     // are consistent with the erasure-meta.
-    pub(crate) fn check_coding_shred(&self, shred: &Shred) -> bool {
+    pub fn check_coding_shred(&self, shred: &Shred) -> bool {
```

Comment:
To minimize the public api surface, instead of making these 2 functions public, can you add a method which takes two shreds and returns `false` if their erasure configs mismatch.
That function can call these 2 functions here internally.